policy/modules/services/matrixd.fc

```diff
@@ -4,3 +4,10 @@
 
 /var/lib/matrix-synapse(/.*)?		gen_context(system_u:object_r:matrixd_var_t,s0)
 /var/log/matrix-synapse(/.*)?		gen_context(system_u:object_r:matrixd_log_t,s0)
+
+ifdef(`distro_gentoo',`
+/etc/synapse(/.*)?			gen_context(system_u:object_r:dhcpc_state_t,s0)
```

Comment:
Isn't this supposed to be labeled `matrixd_conf_t`?

Reply:
oops, and yes, must of had the wrong thing on my clipboard

Reply:
Fixed